js/app/Data/Selection.js

```diff
@@ -132,6 +132,24 @@ define(["app/Class", "app/Events", './TileBounds', "lodash"], function(Class, Ev
       return false;
     },
 
+    hasSelectionInfo: function () {
+      var self = this;
+
+      /* If any of the sortcols contains only negative values, there
+       * is no selection info on the server to be fetched. */
+
```

Comment:
You could use lodash or underscore's `all` and `some` instead of the reduction here. It's clearer than using `reduce` on your own, and faster as well (both `all` and `some` short-circuit in the respective false and true values).

Something like this (untested, only as an example):

```
return _.all(self.sortcols, function(col) {
  return self.data[col] !== undefined 
    && _.any(self.data[col], function(val) { return val >= 0; });
});
```